front/app/components/admin/ContentBuilder/Widgets/ImageMultiloc/index.tsx

```diff
@@ -61,6 +66,9 @@ const Image = ({ imageUrl, alt = {}, dataCode }: Props) => {
       id="e2e-image"
       style={{ pointerEvents: 'none' }}
       minHeight="26px"
+      maxWidth="1150px"
+      margin="0 auto"
```

Comment:
I assume this margin is to center everything? Maybe it's more flexible to let the parent component handle this and make this widget more agnostic about its context?

Reply:
I think it's best these components deal with their own margin and are as context-independent as possible.